src/common/signalk/SKNMEA2000Parser.cpp

```diff
@@ -109,13 +271,143 @@ const SKUpdate& SKNMEA2000Parser::parse128267(const SKSourceInput& input, const
         update->setEnvironmentDepthSurfaceToTransducer(offset);
         update->setEnvironmentDepthBelowSurface(depthBelowTransducer + offset);
       }
+
+      _sku = update;
+      return *_sku;
+    }
+  }
+  DEBUG("Unable to parse N2kMsg with PGN %i", msg.PGN);
+  return _invalidSku;
+}
+
+// *****************************************************************************
+//    129025L: // Position, Rapid Update Lat/Lon
+// *****************************************************************************
+const SKUpdate& SKNMEA2000Parser::parse129025(const SKSourceInput& input, const tN2kMsg& msg, const SKTime& timestamp) {
+  double latitude;
+  double longitude;
+
+  if (ParseN2kPositionRapid(msg, latitude, longitude)) {
+    SKUpdateStatic<2> *update = new SKUpdateStatic<2>();
+    update->setTimestamp(timestamp);
+
+    SKSource source = SKSource::sourceForNMEA2000(input, msg.PGN, msg.Priority, msg.Source);
+    update->setSource(source);
+    update->setNavigationPosition(SKTypePosition(latitude, longitude, 0));
```

Comment:
Note for myself: I think we should use some form of NaN value here for altitude. I am not sure which one but this is something we will have to test later. SignalK tools do not send altitude when they do not get it.